### DISCUSSION TITLE
Is there any quick start document?

### DISCUSSION THREAD

**Discussion by chenliang613** [eclipse-edc/Technology-HuaweiCloud#18] [Q&A]
# Feature Request

_If you are missing a feature or have an idea how to improve this project that should first be discussed, please feel
free to open up a discussion._

## Which Areas Would Be Affected?

_e.g., DPF, CI, build, transfer, etc._

## Why Is the Feature Desired?

_Are there any requirements?_

## Solution Proposal

_If possible, provide a (brief!) solution proposal._

**Comment by mspiekermann** [comment]
Hey @chenliang613, good question! The setup of the repository is still ongoing after the successful adoption request and initial transfer of the sources. We are working on that!

In the meantime, we can use this discussion to collect a list of information that would be required. So, if anyone has a proposal for a structure or want to add ideas for specific points, feel free to comment!

**Reply by liugddx** [reply]
> Hey @chenliang613, good question! The setup of the repository is still ongoing after the successful adoption request and initial transfer of the sources. We are working on that!
> 
> In the meantime, we can use this discussion to collect a list of information that would be required. So, if anyone has a proposal for a structure or want to add ideas for specific points, feel free to comment!

Any progress? I'm excited about this project, but I don't know how to get started with it

**Reply by mspiekermann** [reply]
@liugddx there is the EDC's samples repository [here](https://github.com/[eclipse-edc/Samples](https://github.com/eclipse-edc/Samples/tree/main)/tree/main), which should be your entry point for getting started. Integrating the Huawei Cloud extensions are one of the additional topics, we will add as samples soon. As there are multiple ways of doing this integration, we want to come up with good examples  and descriptions and thus need just some more days.

One prerequisite of having proper releases on Maven Central is now accomplished (for the nightly builds) and we can work on the next steps....

**Reply by liugddx** [reply]
> @liugddx there is the EDC's samples repository [here](https://github.com/%5Beclipse-edc/Samples%5D(https://github.com/eclipse-edc/Samples/tree/main)/tree/main), which should be your entry point for getting started. Integrating the Huawei Cloud extensions are one of the additional topics, we will add as samples soon. As there are multiple ways of doing this integration, we want to come up with good examples and descriptions and thus need just some more days.
> 
> One prerequisite of having proper releases on Maven Central is now accomplished (for the nightly builds) and we can work on the next steps....

Thank you for taking the time to reply to me, it really helped me.
